packages/supplementary-contracts/contracts/blacklist/Blacklist.sol

```diff
@@ -0,0 +1,129 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.24;
+
+import "@openzeppelin/contracts/access/AccessControlEnumerable.sol";
```

Comment:
Did we develop these contracts, otherwise, we should add links to the source.

Reply:
Added URL to the source